bazel/foreign_cc/BUILD

```diff
@@ -118,6 +118,7 @@ envoy_cmake_external(
         "ENABLE_LIB_ONLY": "on",
         "CMAKE_BUILD_TYPE": "RelWithDebInfo",
         "CMAKE_INSTALL_LIBDIR": "lib",
+        "CMAKE_CXX_COMPILER_FORCED": "on",
```

Comment:
Shouldn't we set `CMAKE_C_COMPILER_FORCED` set as well?

Reply:
I didn't add it because it wasn't needed, but fine to add as well. no strong opinion on this.